code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -415,10 +415,10 @@
 				reagents.trans_to(P, vol_each, transfered_by = usr)
 			return TRUE
 		if(item_type == "bottle")
-			var/obj/item/reagent_containers/cup/bottle/P
+			var/obj/item/reagent_containers/cup/tube/P
 			for(var/i in 1 to amount)
-				P = new/obj/item/reagent_containers/cup/bottle(drop_location())
-				P.name = trim("[name] bottle")
+				P = new/obj/item/reagent_containers/cup/tube(drop_location())
+				P.name = trim("[name] test tube")
```

Comment:
Since bottles now hold 50 units, there is little point in having beakers in lathe when you can just get 50u bottles in chem master.

And tube looks like something that is easier to be created on fly (generated in chem master from nothing) compared to a bottle.